app/ide-desktop/lib/content/src/panic.tsx

```diff
@@ -0,0 +1,87 @@
+/** @file This file defines a component that is responsible for displaying a user-facing message
+ * about an application crash (panic). The component is used as a body of a Toastify toast. */
+
+import * as React from 'react'
+import * as toastify from 'react-toastify'
+
+/** Props for `PanicMessage` component. */
+export interface PanicMessageProps {
+    /** The panic message with stack trace. Usually big multiline text. */
+    message: string
+    /** A callback to trigger application restart. */
+    restart: () => void
+}
+
+/** A component displaying panic message inside a toast. */
+export function PanicMessage(props: PanicMessageProps) {
+    const reportUrl =
+        'https://github.com/enso-org/enso/issues/new' +
+        '?labels=--bug%2Ctriage' +
+        '&template=bug-report.yml'
+    return (
+        <div className="flex flex-col">
+            <h1 className="text-xl font-semibold">Enso has crashed.</h1>
+            <p className="mt-3">
+                Enso has encountered a critical error and needs to be restarted. This is a bug, and
+                we would appreciate it if you could report it to us.
+            </p>
+            <p>Please include following panic message in your report:</p>
```

Comment:
ideally the appropriate form input would be pre-filled, but i guess that might not be something github supports.

Reply:
It is possible to prefill the text fields with url params. I didn't managed to pre-select an dropdown option though, but that's not really necessary. The only worry is the potential to generate huge URLs, since the whole panic stacktrace must be contained in a query param. I ended up limiting the allowed stacktrace length to 5000 characters, cutting it off at the line boundary such that it meets the limit. Also took this opportunity to pre-fill the enso version.